app/Http/Controllers/LockerClaimController.php

```diff
@@ -171,7 +171,7 @@ public function end(string $lockerGuid, int $claimId, LockerEndRequest $request)
 
         try {
             $lockerKey->attempt($lockerClaim);
-        } catch (LockerKeyException $e) {
```

Comment:
Due to a lack of exception catching, [LockerException](https://github.com/b-locker/api/blob/2590f1406e690829f5e4e335b590a3551fe0ba5c/app/Exceptions/Models/LockerException.php) (and other exceptions if thrown) would cause a 500 Internal Server Error. That's fixed by making the catch more general.

Another option would be chaining another catch statement.